### ISSUE TITLE
After applying reflow contents inside tree control and data in right pane of "Metrics" page gets truncated: A11y_.NetAspireDashboard_Metrics_Metrics_Reflow

### ISSUE THREAD

**Issue by PoojaNamde** [dotnet/aspire#3393]
"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**

Tags:
#A11yTCS; #A11yMAS; #A11ySev2; #Win11; #ChromiumEdge; #FTP; #DesktopWeb; #.NET Core; #SH_.Net Aspire Dashboard_Web_Mar2024; #WCAG1.4.10; #Reflow; #A11yWCAG2.1;

### Environment Details:
Application Name: .Net Aspire Dashboard
URL: https://localhost:19888/
Chrome Version 122.0.2365.80 (Official build) (64-bit)
Window Version: Win 11
OS Build: 22621.3085

### Prerequisites:

1. Open the web page / screen of the web application that is to be tested
2. Open Chrome/Edge dev Inspect tools. Keyboard shortcut "Ctrl+Shift+I"
3. Ensure devtools are customized to Dock location: "undock into separate window". Minimize devtools so they are not obscuring the page.
4. Zoom the browser window of the page to be tested to 200%.
5. Adjust the browser window's height and width to the required viewport of 320px*256px. As long as devtools remain open while resizing the viewport size will be displayed in the upper right corner of the browser so the size can be verified.

**Alternatively:**

1. Change the OS display settings to a resolution of 1280x1024 at 100% dpi scaling.
2. Open the web page / screen of the web application that is to be tested.
3. Adjust the browser window’s zoom level to 400%.
4. Set the browser to full screen. Keyboard shortcut "F11" key.
5. If any browser sidebars are still visible (e.g., the Edge discover sidebar), these must be closed.

### Repro Steps:

1. Hit the URL: http://localhost:15888/.
2. TAB to "Metrics" tab item and hit ENTER key.
3. Observe that after applying reflow contents inside tree control and data in right pane of "Metrics" page gets truncated.

### Actual Results:
After applying reflow contents inside tree control and data in right pane of "Metrics" page gets truncated

### Expected Results:
After applying reflow contents inside tree control and data in right pane of "Metrics" page should not be truncating and should be visible clearly.

### User Impact:
Low vision users will not be able to see the data present in the page clearly and will miss the information about it.

### Attachment:
![3393_A11y_ NetAspireDashboard_Metrics_Metrics_AfterReflow](https://github.com/dotnet/aspire/assets/90901519/bc06eeee-f107-456d-9670-d636e02a5373)
![3393_A11y_ NetAspireDashboard_Metrics_Metrics_beforeReflow](https://github.com/dotnet/aspire/assets/90901519/3096dd11-2da0-4f96-9e41-e3af007ded50)

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:09-09-24;

Verified the bug in environment
Bug is fixed and does not repro. Hence closing the bug.
Please find the closing attachment
[Video 9-09 at 17.44.webm](https://github.com/user-attachments/assets/9fde44c1-cebe-4403-90cf-96f5ac09dcf3)
